docs/index.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: GradPad
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-T09-1/tp/actions)
+[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2021S1-CS2103T-T09-1/tp)
```

Comment:
Updated these 2 badges so that when you click on them, they take you to our repo's CI and codecov pages respectively: 
![Screenshot 2020-09-27 at 12 22 46 PM](https://user-images.githubusercontent.com/31149701/94356011-2801f280-00bc-11eb-962b-28bae3e689e5.png)